package.json

```diff
@@ -10,15 +10,16 @@
     "dev": "cross-env NODE_ENV=development webpack-dev-server --color --progress",
     "build": "cross-env NODE_ENV=production webpack --color --progress",
     "start": "serve dist",
-    "lint": "eslint --ext .tsx src/"
+    "lint": "eslint --ext .tsx src/ --fix"
```

Comment:
can you add an additional script for lint fix instead of editing the `lint` script?

Reply:
yes, will put in next commit